mermithid/processors/TritiumSpectrum/FakeDataGenerator.py

```diff
@@ -103,12 +104,14 @@ def InternalConfigure(self, params):
         self.simplified_scattering_path = reader.read_param(params, 'simplified_scattering_path', '/host/input_data/simplified_scattering_params.txt')
         self.detailed_scatter_spectra_path = reader.read_param(params, 'path_to_detailed_scatter_spectra_dir', '.')
         self.efficiency_path = reader.read_param(params, 'efficiency_path', '')
+        self.final_states_file = reader.read_param(params, 'final_states_file', '')
```

Comment:
configuring the paths is not always required. it depends on the processor configuration. the paths are therefore checked later in the internal configure method.

Reply:
ah true, this check should happen line 115 (right after the read_param for molecular_final_states...), right?

Reply:
in line 179 it is checked whether final states should be included and if so the configured path is opened in line 181. If the file configuration is wrong or missing, there will be an error in InternalConfigure that the path does not exist. The efficiency file and lineshape files are also only opened after the configuration that  determines whether they should be used or not is checked. but no file existence is checked explicitly since trying to open the files already gives an error. should I add checks for the existence of files so that InternalConfigure returns False if they dont exist instead of throwing errors?